src/firmware/boards/robot_stm32h7/io/primitive_executor.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "firmware/app/primitives/primitive_manager.h"
+#include "firmware/app/world/firmware_robot.h"
+
+/**
+ * Initialize the primitive executor a pointer to the world and
+ * the primitive manager
+ *
+ * @param world A pointer to the world
+ * @param primitive_manager A pointer to the primitive manager
```

Comment:
could we explain _how_ these will be used by the executor? Something like `A pointer to the world in which primitives should be executed` and `A pointer to a primitive manager which will be called to execute the primitives`?

Reply:
updated